tests/x11/firefox/firefox_developertool.pm

```diff
@@ -29,6 +29,7 @@ sub run {
     assert_screen('firefox-developertool-gerneral', 30);
     assert_and_click "firefox-developertool-click_element";
     assert_and_click "firefox-developertool-check_element";
+    wait_still_screen();
```

Comment:
Hello @okurz ,
 This is the first time I try to contribute to os-autoinst. I am still getting used to the procedures. Thank you for the helpful tips on using the snapshots and on the module testing. The failing tests on other modules could be because I was trying to stress the machine with other tasks, but I really didn't dig into them. I thought running only the focused test module would be cheating, but since it seems to be a common practice, I'll try to do it instead.  

Just answering your last comment, you really had a good point on that. So much that we've investigated it better and I was able to reproduce the issue locally, and inside a VM. Even 1 second wasn't enough in some cases.

We actually developed a better alternative. I had discussed it with @czerw and Veronika Svecova and we found that we could add another needle as you suggested instead of the timing.

I was trying to resubmit the PR with those changes, but was having some issues with git on that regard, so I create a new one. 

The new pull request is on https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4975 and the needle commit is on https://gitlab.suse.de/bfilho/os-autoinst-needles-sles/commit/903783045e40aabf7c07700b7878346b8e3c2856.